WebdriverJs-Mocha/tests/performance.js

```diff
@@ -0,0 +1,98 @@
+const assert = require('assert');
+const { Builder } = require('selenium-webdriver');
+const SauceLabs = require('saucelabs');
+
+const username = process.env.SAUCE_USERNAME;
+const accessKey = process.env.SAUCE_ACCESS_KEY;
+const saucelabs = new SauceLabs({
+	username,
+	password: accessKey,
+});
+
+const server = `http://${username}:${accessKey}@ondemand.saucelabs.com:80/wd/hub`;
+const capabilities = {
+	platform: 'OS X 10.10',
+	version: '70.0',
+	browserName: 'chrome',
+	extendedDebugging: true,
+	crmuxdriverVersion: 'stable',
+	name: 'Performance Testing',
+	loggingPrefs: {
+		driver: 'ALL',
+		browser: 'ALL',
+		performance: 'ALL',
+		server: 'ALL',
+	},
```

Comment:
loggingPrefs and chromOptions are not necessary.  We set them in the chef by default when users set`extendedDebugging: true,`